integration-tests/swiss-knife/src/main/scala/swissknife/FlinkCounter.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package swissknife
+
+import cloudflow.flink.FlinkStreamlet
+import org.apache.flink.streaming.api.scala._
+import cloudflow.streamlets.StreamletShape
+import cloudflow.streamlets.avro._
+import cloudflow.flink._
+
+class FlinkCounter extends FlinkStreamlet {
```

Comment:
It is called `FlinkCounter` but it performs no counting but instead does the same transformation as the Akka "transformation" streamlet.

I suggest we use consistent naming for all three transformation streamlets